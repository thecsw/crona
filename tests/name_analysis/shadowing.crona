a : byte array[10];

main : void (a: int array[10], b : int) {
    a[0] = b;
}