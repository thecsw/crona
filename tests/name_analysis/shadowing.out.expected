a(byte array [10]):byte array[10];
main(int array [10],int->void):void(a(int array [10]):int array[10], b(int):int){
	a(int array [10])[0] = b(int);
}